setupext.py

```diff
@@ -1817,8 +1811,6 @@ def add_flags(self, ext):
             (tcl_lib_dir, tcl_inc_dir, tcl_lib,
              tk_lib_dir, tk_inc_dir, tk_lib) = result
             ext.include_dirs.extend([tcl_inc_dir, tk_inc_dir])
-            ext.library_dirs.extend([tcl_lib_dir, tk_lib_dir])
-            ext.libraries.extend([tcl_lib, tk_lib])
```

Comment:
Well, we still need the definitions for these data types:

```
    Tcl_Interp
    Tcl_Command
    Tk_PhotoHandle
    Tk_PhotoImageBlock
    Tk_Window
    Tcl_CmdProc
    ClientData
    Tcl_CmdDeleteProc
```

I guess we could copy those out of some TCL / Tk headers and then be entirely free of the system TCL / Tk.